### PR TITLE
[BRAN-643]-Added-SSM-step-into-CI-to-dev

### PR DESCRIPTION
**Pull request by darkares23**
## Purpose/Description:

Push `src/colors/collage.ts` colors as env variable to AWS SSM 

## Jira Link:

[BRAN-643](https://collagegroup.atlassian.net/browse/BRAN-643?atlOrigin=eyJpIjoiZmUwNmFmMmIwYWJmNDBiYjljOTUwMGZjZmJlOTk3Y2MiLCJwIjoiaiJ9)

## Test Plan:
- Merge to dev and validate if the AWS store manager has the new env variable

[BRAN-643]: https://collagegroup.atlassian.net/browse/BRAN-643?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by Alderr** [comment]
Just in case, you should also include UI colors. We use both often. @darkares23

**Comment by baugusli** [comment]
@Alderr is that the `ui.ts`?

**Comment by baugusli** [comment]
@darkares23 I think we can then just cat both file into one string and send it to SSM store? The question is then when we retrieve it in the backend, it would be in JS format, if you will, then we have to convert it to python or do some string manipulation?

**Comment by darkares23** [comment]
> @darkares23 I think we can then just cat both file into one string and send it to SSM store? The question is then when we retrieve it in the backend, it would be in JS format, if you will, then we have to convert it to python or do some string manipulation?

Working in combining the 2 files into one json, so the backend can convert it to an object using `parse()`

File: scripts/combine-json.sh [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+json_collage=$(sed '/export default/d' src/colors/collage.ts | jq -c .)
+json_ui=$(sed '/export default/d' src/colors/ui.ts | jq -c .)
+
+# Combine JSON
+combined_json="{\"collage\": $json_collage, \"ui\": $json_ui}"
+
+echo "{ \"collage\": $json_collage, \"ui\": $json_ui }"
+
+# Update AWS ssm parameter
+aws ssm put-parameter --name "/config/COLLAGE_UI_COLORS" --type "String" --value "$combined_json" --overwrite
```
Comment (Alderr) [inline scripts/combine-json.sh]: EOL

**Review by Alderr — APPROVED** [review]

**Pull request merged**